build.gradle

```diff
@@ -32,4 +32,5 @@ if (project.hasProperty('release.useLastTag')) {
 
 test{
      maxHeapSize = "2g"
+     maxParallelForks = 8
```

Comment:
This could increase the likelihood time-sensitive tests fail due to higher server utilization.